### DISCUSSION TITLE
Release 0.23.0-alpha3 🏗️ Work in progress 🏗️  stacks are back

### DISCUSSION THREAD

**Discussion by simulot** [simulot/immich-go#523] [Announcements]
The code refactoring continue. Better code, less bugs, few features...

### New features

- `--manage-burst=BurstFlag`   Manage burst photos. Possible values are: 
  - `StackKeepRaw` Discard JPEG files, and stack the RAW files (default)
  - `StackKeepJPEG` Discard RAW files, and stack the JPEG files 
  - `Stack` Stack all photos, RAW and JPEG photos are imported in the same stack
      
- `--manage-heic-jpeg=HeicJpgFlag` Manage coupled HEIC and JPEG files. Possible values: 
  - `KeepHeic` Keep only the HEIC files (default)
  - `KeepJPG` Keep only the JPEG files
  - `StackCoverHeic` Stack both, the HEIC file is the cover
  - `StackCoverJPG` Stack both, the JPEG file is the cover

- `--manage-raw-jpeg=RawJPGFlag`   Manage coupled RAW and JPEG files. Possible values: 
  - `KeepRaw` Keep only the RAW files (default)
  - `KeepJPG` Keep only the JPEG files
  - `StackCoverRaw` Stack both, the RAW file is the cover
  - `StackCoverJPG` Stack both, the JPEG file is the cover

**Full Changelog**: https://github.com/simulot/immich-go/compare/0.21.2...0.23.0-alpha3

<hr /><em>This discussion was created from the release <a href='https://github.com/simulot/immich-go/releases/tag/0.23.0-alpha3'>Release 0.23.0-alpha3 🏗️ Work in progress 🏗️  stacks are back</a>.</em>

**Comment by bpmich** [comment]
Love the expeditious repairs, been looking for a fix since 1.118.0 release. Any more info on how this command will work on existing libraries? Just finished crunching all the jobs in Immich before I realized stacking was an issue. Hoping to avoid another 650GB upload 🙄

**Reply by simulot** [reply]
yes, the intention is to give a set of tools for working on immich photos for stacking bursts, handling jpg/raw, and more.
So, you wont have to start over your installation
